src/NHibernate/Persister/Entity/AbstractEntityPersister.cs

```diff
@@ -2512,20 +2512,27 @@ protected IUniqueEntityLoader CreateEntityLoader(LockMode lockMode)
 		}
 
 		protected bool Check(int rows, object id, int tableNumber, IExpectation expectation, DbCommand statement)
+		{
+			return Check(rows, id, tableNumber, expectation, statement, false);
+		}
+
+		protected bool Check(int rows, object id, int tableNumber, IExpectation expectation, DbCommand statement, bool forceThrowStaleException = false)
```

Comment:
it hides the other method. Please remove the default value.

Reply:
Intention is to get rid of other method in next major. I just forgot to add TODO about it.

Reply:
>  forgot to add TODO

Fixed